bindings/go/evmc/evmc_test.go

```diff
@@ -59,3 +59,9 @@ func TestExecuteEmptyCode(t *testing.T) {
 		t.Errorf("execution returned unexpected error: %v", err)
 	}
 }
+
+func TestRevision(t *testing.T) {
+	if MaxRevision != London {
```

Comment:
Should there also be a test for `LatestStableRevision` here?

Reply:
Added.